tests/adapter/dbt/tests/adapter/utils/base_utils.py

```diff
@@ -2,9 +2,11 @@
 from dbt.tests.util import run_dbt
 
 macros__equals_sql = """
-{% macro equals(actual, expected) %}
-{# -- actual is not distinct from expected #}
-(({{ actual }} = {{ expected }}) or ({{ actual }} is null and {{ expected }} is null))
+{% macro equals(expr1, expr2) -%}
+case when (({{ expr1 }} = {{ expr2 }}) or ({{ expr1 }} is null and {{ expr2 }} is null))
+    then 0
+    else 1
+end = 0
```

Comment:
`1`s and `0`s to be platform agnostic?

Reply:
Yep, you got it!

Markus Winand explains it nicely [here](https://modern-sql.com/feature/is-distinct-from#conforming-alterantives).

The SQL standard way to express this is:
```
a is not distinct from b
```

But not all databases support that syntax.

Reply:
Oh no
> This is often not a problem because [SQL generally treats unknown like false when making binary decisions](https://modern-sql.com/concept/three-valued-logic#general-rule) such as accepting or rejecting a row for a where clause.